geomesa-tools/examples/ingest/csv/example.csv

```diff
@@ -1,4 +1,4 @@
-ID,Name,Age,LastSeen,Friends,Lon,Lat,Vis
-23623,Harry,20,2015-05-06,"Will, Mark, Suzan","patronus->10,expelliarmus->9",-100.236523,23,user
+ID,Name,Age,LastSeen,Friends,Skills,Lon,Lat,Vis
+23623,Harry,20,2015-05-06,"Will, Mark, Suzan","patronus->10,expelliarmus->9",-100.2365,23,user
```

Comment:
The inferencer can figure out maps?!

Reply:
no haha, it just treats it as a string - but the header row in this file didn't match the column count, so the names were off when auto-ingested